packages/sessions-sdk-react/src/components/send-token-page.tsx

```diff
@@ -77,7 +78,10 @@ export const SendTokenPage = ({
         )
         .then((result) => {
           if (result.type === TransactionResultType.Success) {
-            toast.success("Tokens sent successfully!");
+            toast.success(
+              "Tokens sent successfully!",
```

Comment:
if the relevant paymaster doesn't have preflight might this tx fail on chain? in which case tokens were not actually sent